app/root.css

```diff
@@ -488,6 +488,9 @@ footer > *:not(:last-child) {
   color: inherit;
   font-style: italic;
 }
+.result-item.none-of-the-above button {
+  all: unset; /* Make the button look like a span */
+}
```

Comment:
for lack of a better solution and because my css skills are rudimentary at best

Reply:
can you reuse existing class for `.transparent-button`, please?

Reply:
perfect! Just what I was looking for - added